docs/getting_started.md

```diff
@@ -0,0 +1,46 @@
+## Getting started
+
+you can get a basic react system up and running extremely quickly by following these steps.
+
+1. Create an NPM account, if you don't have one and get an NPM token.
+
+2. Export your NPM token to the environment variable NPM_TOKEN.
+
+3. Make a directory/folder with the name of your project.
+
+4. navigate to the directory/folder you've just created.
+
+5. Run this command: `npx @amidostacks/scaffolding-cli@1.20.0 run -i`
```

Comment:
Remove the version, should pull in latest.

Reply:
changed to latest